### ISSUE TITLE
axis font not poppins

### ISSUE THREAD

**Issue by MrichardsPSRC** [psrc/psrcplot#13]
Using static_bar_chart function generates:
![image](https://user-images.githubusercontent.com/72169299/211077162-ce054e1e-2cba-4b4d-a885-3263ed13cab8.png)

All of the font is poppins except for the text corresponding to the y argument:
```
own_race_chart <- static_bar_chart(t=dt_2021,
                                   x="share", y="HRACE",
                                   fill="HRACE",
                                   pos="dodge",
                                   est="percent",
                                   # moe="share_moe",
                                   # href="ref_val", hrefnm="Region",
                                   # hrefcl='#00716c',
                                   title="Homeownership by Race and Hispanic/Latinx Origin",
                                   subtitle=NULL,
                                   source="2021 American Commmunity Survey 1-year PUMS Data",
                                   alt=NULL,
                                   xlabel=NULL, ylabel=NULL,
                                   dec = 2, color="psrc_light") + theme(legend.position = "none") + scale_x_discrete(limits=rev) #alphabetical order
```

**Comment by Ennazus** [comment]
I'm stuck on this one; I tried to add to psrc_styles:     axis.title = ggplot2::element_text(size=12, family =font, color="#2f3030"),
    axis.text = ggplot2::element_text(size=11, family=font, color="#2f3030"),
that didn't work; 


I tried to reassert the style at the end of the plot creation: c <- c + psrcplot::psrc_style(); that didn't work

**Comment by jensenmj** [comment]
Solved with commit 938aa65.

**Comment by jensenmj** [comment]
(This only applies to static charts, as interactives currently remove and rebuild the labels.)
